subsys/bluetooth/host/l2cap.c

```diff
@@ -83,8 +83,14 @@ struct bt_l2cap {
 	struct bt_l2cap_le_chan	chan;
 };
 
+static struct bt_l2cap_cb *l2cap_callbacks;
```

Comment:
Call the variable `ecred_cb` (assuming you agree with my other naming suggestions)